[Response Ops] [Rule Form] Remove V1 Rule Form Flyout

Zacqary opened the pull request:
## Summary

Closes #195211 

Removes all old rule form flyout code, which should no longer be referenced at all after https://github.com/elastic/kibana/pull/206685/

### Checklist

- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios





<!--ONMERGE {"backportTargets":["9.0"]} ONMERGE-->

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/272433)
* Commit: fc81cb4e8ed6a6655886383a5d46c9c39acfae6c

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/272433#0194be14-37a3-41ee-9b0b-5ed43e7e86df)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/272433","number":272433,"commit":"fc81cb4e8ed6a6655886383a5d46c9c39acfae6c"}],"number":272433}
buildkite-pr-comment-->

Commit:
Remove V1 Rule Form flyout

# Conflicts:
#	x-pack/platform/plugins/shared/triggers_actions_ui/.storybook/decorator.tsx
#	x-pack/platform/plugins/shared/triggers_actions_ui/common/experimental_features.ts
#	x-pack/platform/plugins/shared/triggers_actions_ui/public/common/get_experimental_features.test.tsx

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/styled_components_mapping'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281160)
* Commit: 3e0313f7963a5609e892c1acb5c476217d3a0f85

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/281160#01956709-50ad-48a7-a0ce-d30935c9300f)

### History
* :broken_heart: [Build #272433](https://buildkite.com/elastic/kibana-pull-request/builds/272433) failed fc81cb4e8ed6a6655886383a5d46c9c39acfae6c

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281160","number":281160,"commit":"3e0313f7963a5609e892c1acb5c476217d3a0f85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/272433","number":272433,"commit":"fc81cb4e8ed6a6655886383a5d46c9c39acfae6c"}],"number":281160}
buildkite-pr-comment-->

Commit:
Remove monitoring rule form test

Commit:
Fix i18n, restore rule form  route, form route, example components

Commit:
Merge branch '195211-v1-flyout-remove' of https://github.com/Zacqary/kibana into 195211-v1-flyout-remove

Commit:
Remove double import of RuleFormRoute

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/yarn_deduplicate'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281259)
* Commit: d33aff8291051be3000182a2a02248f07f9da24c

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/281259#01956847-cad1-4fcf-ba2f-791d137127fd)

### History
* :broken_heart: [Build #281160](https://buildkite.com/elastic/kibana-pull-request/builds/281160) failed 3e0313f7963a5609e892c1acb5c476217d3a0f85
* :broken_heart: [Build #272433](https://buildkite.com/elastic/kibana-pull-request/builds/272433) failed fc81cb4e8ed6a6655886383a5d46c9c39acfae6c

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281259","number":281259,"commit":"d33aff8291051be3000182a2a02248f07f9da24c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281160","number":281160,"commit":"3e0313f7963a5609e892c1acb5c476217d3a0f85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/272433","number":272433,"commit":"fc81cb4e8ed6a6655886383a5d46c9c39acfae6c"}],"number":281259}
buildkite-pr-comment-->

Commit:
Remove unused exports

Commit:
Merge branch '195211-v1-flyout-remove' of https://github.com/Zacqary/kibana into 195211-v1-flyout-remove

Commit:
Merge remote-tracking branch 'upstream/main' into 195211-v1-flyout-remove

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281575)
* Commit: 6352f0fd9b525c132c110a39cae4d3d970c1cd0e

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/281575#01956ceb-a73b-46b2-a6aa-aee6ef81020e)

### History
* :broken_heart: [Build #281259](https://buildkite.com/elastic/kibana-pull-request/builds/281259) failed d33aff8291051be3000182a2a02248f07f9da24c
* :broken_heart: [Build #281160](https://buildkite.com/elastic/kibana-pull-request/builds/281160) failed 3e0313f7963a5609e892c1acb5c476217d3a0f85
* :broken_heart: [Build #272433](https://buildkite.com/elastic/kibana-pull-request/builds/272433) failed fc81cb4e8ed6a6655886383a5d46c9c39acfae6c

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281575","number":281575,"commit":"6352f0fd9b525c132c110a39cae4d3d970c1cd0e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281259","number":281259,"commit":"d33aff8291051be3000182a2a02248f07f9da24c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281160","number":281160,"commit":"3e0313f7963a5609e892c1acb5c476217d3a0f85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/272433","number":272433,"commit":"fc81cb4e8ed6a6655886383a5d46c9c39acfae6c"}],"number":281575}
buildkite-pr-comment-->

Commit:
Fix typecheck

elasticmachine commented:
Pinging @elastic/response-ops (Team:ResponseOps)

pickypg reviewed (approved):
LGT Stack Monitoring

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/281634)
* Commit: a732babcc891297c9a9d718e030d0d3e52ccfef4
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-209171)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 860 | 835 | -25 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `@kbn/response-ops-rule-form` | 182 | 183 | +1 |
| `triggersActionsUi` | 529 | 524 | -5 |
| total |  |  | -4 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 1.5MB | 1.5MB | -99.7KB |

#### Public APIs missing exports
> Total count of every type that is part of your API that should be exported but is not. This will cause broken links in the API documentation system. Target amount is 0. `Run node scripts/build_api_docs --plugin [yourplugin] --stats exports` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `@kbn/response-ops-rule-form` | 6 | 7 | +1 |
| `triggersActionsUi` | 38 | 36 | -2 |
| total |  |  | -1 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 109.7KB | 108.7KB | -1.0KB |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `@kbn/response-ops-rule-form` | 183 | 184 | +1 |
| `triggersActionsUi` | 537 | 530 | -7 |
| total |  |  | -6 |

#### async chunk count
| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 64 | 54 | -10 |

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 119 | 102 | -17 |

#### References to deprecated APIs
| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 10 | 6 | -4 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/87e7cd94d1d649596dc0f23bf4cf730704fb4845) | [after](https://github.com/elastic/kibana/commit/a732babcc891297c9a9d718e030d0d3e52ccfef4) | diff |
| --- | --- | --- | --- |
| `triggersActionsUi` | 124 | 107 | -17 |


</details>


### History
* :broken_heart: [Build #281575](https://buildkite.com/elastic/kibana-pull-request/builds/281575) failed 6352f0fd9b525c132c110a39cae4d3d970c1cd0e
* :broken_heart: [Build #281259](https://buildkite.com/elastic/kibana-pull-request/builds/281259) failed d33aff8291051be3000182a2a02248f07f9da24c
* :broken_heart: [Build #281160](https://buildkite.com/elastic/kibana-pull-request/builds/281160) failed 3e0313f7963a5609e892c1acb5c476217d3a0f85
* :broken_heart: [Build #272433](https://buildkite.com/elastic/kibana-pull-request/builds/272433) failed fc81cb4e8ed6a6655886383a5d46c9c39acfae6c

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281634","number":281634,"commit":"a732babcc891297c9a9d718e030d0d3e52ccfef4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281575","number":281575,"commit":"6352f0fd9b525c132c110a39cae4d3d970c1cd0e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281259","number":281259,"commit":"d33aff8291051be3000182a2a02248f07f9da24c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/281160","number":281160,"commit":"3e0313f7963a5609e892c1acb5c476217d3a0f85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/272433","number":272433,"commit":"fc81cb4e8ed6a6655886383a5d46c9c39acfae6c"}],"number":281634}
buildkite-pr-comment-->

pheyos reviewed (approved):
`x-pack/test_serverless/functional/config.base.ts` changes LGTM

andreadelrio reviewed (approved):
Sass file removed.

cnasikas reviewed (approved):
So good to see all these code to get removed 🎉!

kibanamachine commented:
Starting backport for target branches: 9.0

https://github.com/elastic/kibana/actions/runs/13737553995

Zacqary commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/213746">](https://github.com/elastic/kibana/pull/213746)|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/213748">](https://github.com/elastic/kibana/pull/213748)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kibanamachine commented:
Looks like this PR has a backport PR but it still hasn't been merged. Please merge it ASAP to keep the branches relatively in sync.

kibanamachine commented:
Looks like this PR has a backport PR but it still hasn't been merged. Please merge it ASAP to keep the branches relatively in sync.
